### DISCUSSION TITLE
Copy dashboard variable to a new dashboard

### DISCUSSION THREAD

**Discussion by laurielounge** [grafana/grafana#39455] [Dashboards, Panels & Visualizations]
Hi all,

I can copy a dashboard panel to a new dashboard. Why not copy a dashboard variable to a new dashboard?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
